ow_power_system/include/power_system_node.h

```diff
@@ -0,0 +1,46 @@
+#ifndef __POWER_SYSTEM_NODE_H__
+#define __POWER_SYSTEM_NODE_H__
+
+#include <ros/ros.h>
+#include <chrono>
+
+#include "PrognoserFactory.h"
+
+class PowerSystemNode
+{
+public:
+  PowerSystemNode();
```

Comment:
It's good to also be explicit about the copy constructor, assignment operator, and destructor.  I.e. define them or 'delete' them or a comment that you want to use the defaults.

Reply:
Yes, originally there was some initialization in the constructor. But I have moved things around and the constructor is no longer utilized .. I will omit it!

Reply:
addressed